### PR TITLE
Suggested hibernation docs tweaks

### PR DESCRIPTION
**Pull request by ianbuss**
Closes https://github.com/astronomer/laminar/issues/1488

### CODE REVIEW

File: astro/deployment-settings.md [hunk 1]
```diff
@@ -286,6 +286,13 @@ To create a hibernation schedule:
 4. (Optional) Specify additional hibernation schedules for your Deployment.
 5. Select **Update Deployment** to save your changes.
```
Comment (jwitz) [inline astro/deployment-settings.md:288]:
```suggestion
:::tip

You can use the following example cron expressions to implement common Deployment hibernation schedules:

| Schedule | Start schedule | End schedule |
|----------|----------------|---------------|
| Hibernate from 5:00 PM to 9:00 AM| 0 17 * * * | 0 9 * * * |
| Hibernate on weekends (Friday 5:00PM to Monday 9:00AM | 0 17 * * 5 | 0 9 * * 1 |

:::
```

Comment (jwitz) [reply astro/deployment-settings.md:288]: Resolves https://github.com/astronomer/docs/issues/3251

File: astro/deployment-settings.md [hunk 2]
```diff
@@ -286,6 +286,13 @@ To create a hibernation schedule:
...
 
+:::info
+
+Note that all cron schedules are interpreted in the UTC timezone. So, for example, if you want your Deployment to
+hibernate at 1700 EST, you should use `0 22 * * *` as the cron expression.
+
+:::
```
Comment (jwitz) [inline astro/deployment-settings.md:294]:
```suggestion
:::warning

Cron schedules are set in UTC. If you're running a Deployment in another time zone, you must convert the cron expression for your time zone to UTC. For example, if you want your Deployment to hibernate at 17:00 EST, you  use `0 22 * * *` (22:00 UTC) as the cron expression.

:::
```

File: astro/deployment-settings.md [hunk 3]
```diff
@@ -300,17 +307,17 @@ When your hibernation schedule starts:
...
+If you need to run a task or DAG on a Deployment that is currently in hibernation, you can manually wake up a Deployment from hibernation before the end of its schedule.
 
 1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.
 2. Click the **More Actions** menu of the Deployment you want to update, then select **Wake up from Hibernation**.
-3. Select **Remove override and return to normal schedule**, then click **Confirm**. Alternatively, you can adjust the existing manual hibernation period if you don't immediately need to wake up the Deployment.
+3. Select the appropriate option from the dialog: if you need to wake immediately for an indefinite period and ignore any configured schedules, select **Wake until further notice**. If you know how long you want the deployment to be awake, select **Wake until set time and date** and specify a time. If you want to go back to the normal schedules and remove the override, select **Remove override and return to normal schedule**. Once you've made your selection, click **Confirm**.
```
Comment (jwitz) [inline astro/deployment-settings.md:320]:
```suggestion
3. Select one of the following options for how you want your Deployment to wake up:

    - **Wake until further notice**: Your Deployment wakes up immediately for an indefinite period and ignores any configured hibernation schedules. 
    - **Wake until set time and date**: Specify a time that the Deployment should go back into hibernation after waking up. 
    - *Remove override and return to normal schedule**: The Deployment returns to following your configured hibernation schedules.
    4. Click **Confirm**.
```

**Review by jwitz — CHANGES_REQUESTED** [review]
@ianbuss Thank you for these edits! I also threw in some example crons for users, can you check those and let me know if there's any other examples we should add? After that looks good, I'll merge this

**Comment by jwitz** [comment]
Btw `git pull origin main` should resolve the build issues

**Comment by ianbuss** [comment]
@jwitz have updated, please take another look.

File: astro/deployment-settings.md (hunk 2 above)
Comment (jwitz) [reply astro/deployment-settings.md:305]: Bumping bc this suggestion might have gotten lost in the merge

**Review by jwitz — APPROVED** [review]
LGTM! Bumped one suggestion that might have gotten buried from the last review

**Pull request merged**